ml-operator/templates/redis.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
```

Comment:
Is there any reason you are defining your own Redis instance instead of relying on an external Redis instance?

I.e. I would not recommend doing this as you may have to support making this HA, etc.

At the very least one should have the option of deploying this "included" Redis or disabling it and pointing to an external Redis.

Reply:
It's just a simple cache for the image-watcher service (I've added a note to the readme that explains in more detail), and it's state can and should be restored by the image-watcher service. I'll add config to be able to turn it off, and byo redis.